testing/test_likelihoods.py

```diff
@@ -181,14 +181,47 @@ def testSymmetric(self):
             pred = tf.Session().run(l.predict_density(F, F, Y), feed_dict={x: l.get_free_state(), F: F_data})
             variational_expectations = tf.Session().run(l.variational_expectations(F, F, Y),
                                                         feed_dict={x: l.get_free_state(), F: F_data})
-        self.assertTrue(np.allclose(mu, np.ones((nPoints, nClasses))/nClasses, tolerance, tolerance))
-        self.assertTrue(np.allclose(pred, np.ones((nPoints, 1))/nClasses, tolerance, tolerance))
+        expected_mu = (1./nClasses * (1. - epsilon) + (1. - 1./nClasses) * epsilon / (nClasses - 1)) * np.ones((nPoints, 1))
+        self.assertTrue(np.allclose(mu, expected_mu, tolerance, tolerance))
+        expected_log_denisty = np.log(expected_mu)
+        self.assertTrue(np.allclose(pred, expected_log_denisty, 1e-3, 1e-3))
```

Comment:
yes, fine. 

It's always going to be tricky to get fine tolerances when the numbers we're targetting are [0, 1].